core/domain/feedback_jobs_continuous.py

```diff
@@ -135,26 +149,25 @@ def get_thread_analytics_multi(cls, exploration_ids):
                 exploration_ids))
 
         return [feedback_domain.FeedbackAnalytics(
-            exploration_ids[i],
-            (realtime_models[i].num_open_threads
-             if realtime_models[i] is not None else 0) +
+            exploration_ids[i], (realtime_models[i].num_open_threads
```

Comment:
I was having some trouble passing the linter with how it was before, with it saying wrong hanging indentation. I looked into the error a bit and there was some answers on stackoverflow about how this could be caused by tabs/spaces but using only spaces still caused the linter to error.